lib/browser/rpc.js

```
@@ -90,74 +107,74 @@ export function asyncOpenRealm(id, config, callback) {
 }
 
 export function _functionRPC(payload) {
     const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
-    const result = sendRequest('_function', { arguments: args });
+    const result = sendRequest("_function", { arguments: args });
     return deserialize(undefined, result);
 }
 
 export function _googleRPC(authCode) {
     const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
-    const result = sendRequest('_google', { arguments: args });
+    const result = sendRequest("_google", { arguments: args });
     return deserialize(undefined, result);
 }
 export function _userApiKeyRPC(user_key) {
     const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
-    const result = sendRequest('_userAPIKey', { arguments: args });
+    const result = sendRequest("_userAPIKey", { arguments: args });
     return deserialize(undefined, result);
 }
 
 export function _serverApiKeyRPC(server_key) {
     const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
-    const result = sendRequest('_serverAPIKey', { arguments: args });
+    const result = sendRequest("_serverAPIKey", { arguments: args });
     return deserialize(undefined, result);
 }
 
 export function _jwtRPC(token) {
     const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
-    const result = sendRequest('_jwt', { arguments: args });
+    const result = sendRequest("_jwt", { arguments: args });
     return deserialize(undefined, result);
 }
 
 export function callSyncFunction(name, args) {
     args = (args || []).map((arg) => serialize(null, arg));
-    let result = sendRequest('call_sync_function', { name, arguments: args });
+    let result = sendRequest("call_sync_function", { name, arguments: args });
     return deserialize(undefined, result);
 }
 
 export function callMethod(realmId, id, name, args) {
-    if (args) {
+    if (args && Array.isArray(args)) {
```

this should be a argument validation instead of silencing the error